lectures/eigen_II.md

```diff
@@ -65,38 +63,56 @@ We introduced irreducible matrices in the [Markov chain lecture](mc_irreducible)
 
 Here we generalize this concept:
 
-An $n \times n$ matrix $A$ is called irreducible if, for each $i,j$ with $1 \leq i, j \leq n$, there exists a $k \geq 0$ such that $a^{k}_{ij} > 0$.
-
-A matrix $A$ that is not irreducible is called reducible.
+Let $a^{k}_{ij}$ be element $(i,j)$ of $A^k$.
 
-Here are some examples to illustrate this further.
+An $n \times n$ nonnegative matrix $A$ is called irreducible if $A + A^2 + A^3 + \cdots \gg 0$, where $\gg 0$ denotes every element in $A$ is nonnegative.
 
-1. $A = \begin{bmatrix} 0.5 & 0.1 \\ 0.2 & 0.2 \end{bmatrix}$ is irreducible since $a_{ij}>0$ for all $(i,j)$.
+In other words, for each $i,j$ with $1 \leq i, j \leq n$, there exists a $k \geq 0$ such that $a^{k}_{ij} > 0$.
 
-2. $A = \begin{bmatrix} 0 & 1 \\ 1 & 0 \end{bmatrix}$ is irreducible since $a_{12},a_{21} >0$ and $a^{2}_{11},a^{2}_{22} >0$.
+Here are some examples to illustrate this further:
 
-3. $A = \begin{bmatrix} 1 & 0 \\ 0 & 1 \end{bmatrix}$ is reducible since $A^k = A$ for all $k \geq 0$ and thus
-   $a^{k}_{12},a^{k}_{21} = 0$ for all $k \geq 0$.
+$$
+A = \begin{bmatrix} 0.5 & 0.1 \\ 
+                    0.2 & 0.2 
+\end{bmatrix}
+$$
 
-### Primitive matrices
+$A$ is irreducible since $a_{ij}>0$ for all $(i,j)$.
 
-Let $A$ be a square nonnegative matrix and let $A^k$ be the $k^{th}$ power of $A$.
+$$
+B = \begin{bmatrix} 0 & 1 \\ 
+                    1 & 0 
+\end{bmatrix}
+, \quad
+B^2 = \begin{bmatrix} 1 & 0 \\ 
+                      0 & 1
+\end{bmatrix}
+$$
 
-A matrix is called **primitive** if there exists a $k \in \mathbb{N}$ such that $A^k$ is everywhere positive.
+$B$ is irreducible since $b_{12},b_{21} >0$ and $b^{2}_{11},b^{2}_{22} >0$.
 
-It means that $A$ is called primitive if there is an integer $k \geq 0$ such that $a^{k}_{ij} > 0$ for *all* $(i,j)$.
+$$
+C = \begin{bmatrix} 1 & 0 \\ 
+                    0 & 1 
+\end{bmatrix}
+$$
 
-We can see that if a matrix is primitive, then it implies the matrix is irreducible.
+$C$ is reducible since $C^k = C$ for all $k \geq 0$ and thus
+   $c^{k}_{12},c^{k}_{21} = 0$ for all $k \geq 0$.
 
 ### Left eigenvectors
 
-We previously discussed right (ordinary) eigenvectors $Av = \lambda v$.
+Recall that we previously discussed eigenvectors in {ref}`Eigenvalues and Eigenvectors <la_eigenvalues>`.
 
-Here we introduce left eigenvectors.
+The eigenvalue $\lambda$ and its cooresponding eigenvecotr $v$ of matrix $A$ satisfy
 
-Left eigenvectors will play important roles in what follows, including that of stochastic steady states for dynamic models under a Markov assumption.
+$$
+Av = \lambda v.
+$$
+
+In this section we will define $v$ as right eigenvectors since we will be introducing left eigenvectors now.
```

Comment:
In this section we introduce left eigenvectors.

To avoid confusion, what we previously referred to as "eigenvectors" will be called "right eigenvectors".